src/importexport/midiimport/internal/midiimport/importmidi_simplify.cpp

```diff
@@ -41,7 +41,7 @@ bool areDurationsEqual(
         sum += ReducedFraction(d.second.fraction()) / d.first;
     }
 
-    return desiredLen == desiredLen;
+    return sum == desiredLen;
```

Comment:
See also #7179 for 3.x, and the problems that went along with it, 2 mtests crashing (tests that are not enabled yet in master IIRC)